docs/usage.md

```diff
@@ -1,66 +1,49 @@
 # nf-core/staramr: Usage
 
-## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/staramr/usage](https://nf-co.re/staramr/usage)
-
-> _Documentation of pipeline parameters is generated automatically from the pipeline schema and can no longer be found in markdown files._
-
 ## Introduction
 
-<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
+This is the [nf-core](https://nf-co.re/)-based pipeline for [StarAMR](https://github.com/phac-nml/staramr). This pipeline scans bacterial genome contigs against the ResFinder, PointFinder, and PlasmidFinder databases and compiles a summary report of detected antimicrobial resistance genes.
 
 ## Samplesheet input
 
-You will need to create a samplesheet with information about the samples you would like to analyse before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row as shown in the examples below.
+You will need to create a samplesheet with information about the samples you would like to analyze before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 2-3 columns, and a header row as shown in the examples below.
 
 ```bash
 --input '[path to samplesheet file]'
 ```
 
-### Multiple runs of the same sample
-
-The `sample` identifiers have to be the same when you have re-sequenced the same sample more than once e.g. to increase sequencing depth. The pipeline will concatenate the raw reads before performing any downstream analysis. Below is an example for the same sample sequenced across 3 lanes:
-
-```console
-sample,fastq_1,fastq_2
-CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L003_R1_001.fastq.gz,AEG588A1_S1_L003_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L004_R1_001.fastq.gz,AEG588A1_S1_L004_R2_001.fastq.gz
-```
-
 ### Full samplesheet
 
-The pipeline will auto-detect whether a sample is single- or paired-end using the information provided in the samplesheet. The samplesheet can have as many columns as you desire, however, there is a strict requirement for the first 3 columns to match those defined in the table below.
+The input samplesheet requires two columns: `sample`,`contigs` with an optional third column `species`. The `species` column is used in the selecting of the Pointfinder organism database (empty if "None"). Rows of the `sample` column within a samplesheet must be unqiue. All other columns will be ignored.
```

Comment:
I'm not quite sure what the final sentence **All other columns will be ignore** means. As in uniqueness is ignored?

Reply:
I'd taken it from `speciesabundance/doc` [here](https://github.com/phac-nml/speciesabundance/blob/main/docs/usage.md#full-samplesheet), I took it to mean that the `schema_input.json` only uses the first three columns in the pipeline (`sample`, `contigs`, `species`). Other columns would be meaningless to the pipeline. I tested this with a samplesheet that has a fourth column. It doesn't crash the pipeline but simply is not used.

Reply:
Okay, thanks. Could you change this to make it more clear. Something like

> Any additional columns that aren't named `sample`, `contigs`, or `species` will be ignored by the pipeline

Reply:
Changed it on `README.md` and `doc/usage.md` d3b3ac8